test/rounding.jl

```diff
@@ -495,3 +495,11 @@ end
         end
     end
 end
+
+@testset "Rounding to Zero with RoundFromZero" begin
+    @testset "Testing float types: $f" for f ∈ (Float16, Float32, Float64, BigFloat)
+        @testset "Testing value types: $t" for t ∈ (Bool, Rational{Int8})
+            @test iszero(f(zero(t), RoundFromZero))
+        end
+    end
+end
```

Comment:
I'd add a few tests that aren't rounding zero to zero:
```suggestion
    end
    @test Float16(100000, RoundToZero) === floatmax(Float16)
    @test Float16(100000, RoundFromZero) === Inf16
    @test Float16(-100000, RoundToZero) === -floatmax(Float16)
    @test Float16(-100000, RoundFromZero) === -Inf16
    @test Float32(nextfloat(0.0), RoundFromZero) === nextfloat(0.0f0)
end
```